source/uwp/SharedRenderer/lib/ActionHelpers.cpp

```diff
@@ -764,12 +764,13 @@ namespace AdaptiveCards::Rendering::Xaml_Rendering::ActionHelpers
 
                 case winrt::FallbackType::Content:
                     action = action.FallbackContent();
+                    actionType = action.ActionType();
```

Comment:
still needed? if not, remove:)

Reply:
Yup, we need this so we know what kind of button to create in 'CreateActionButtonInActionSet'